### DISCUSSION TITLE
Stream links updated and they don't play?

### DISCUSSION THREAD

**Discussion by ThetaX55** [iptv-org/community#537] [Q&A]
Looking at the streams part. 
It's updated. But a ton of them don't work. 
What's the point? Are those on some of the time? Something i missed?

**Comment by PopeyeTheSai10r** [comment]
Hello @ThetaX55, If you find streams that are no longer valid, you can [report them](https://github.com/iptv-org/iptv/issues/new/choose) using Report broken stream so they get removed.

Please keep in mind that some channels are `Geo-blocked` or `Not 24/7`

**Comment by ThetaX55** [comment]
Invited buddy's over and showed them these.
The one's posted on this dude's gethub.
They seem to be almost same channels you get on the smart TV for free. or on tv plans basic channels/
Pluto TV my buddy says is a free app where he is from. As a example.
We started to think those channels on cable have added content. Because some say stay tuned or off air at times wile on cable they put some stuff there.
Cable, telephone and internet is all together. Kinda weird it's there.
Maybe certain locations don't get it those channels.
Do i get better results really with a VPN? CHannels that don't seem to work. I don't even want to bother. we already pay for the package. Don't get why i have to pay more to use another location internet and why i don't just pick it up.

**Reply by Jitterdoomer** [reply]
Good news is that Stream-Link's channels are available in all countries except for Mainland China and Taiwan. I tested Denmark's VPN and it streamed any channels regularly without an error.
